spec/lib/bolt_spec/integration.rb

```diff
@@ -11,12 +11,22 @@ def run_cli(arguments)
       allow(cli).to receive(:outputter).and_return(outputter)
 
       opts = cli.parse
-      cli.execute(opts)
+
+      if rescue_exec
+        err_kls = error_support ? Bolt::Error : StandardError
+        begin
+          cli.execute(opts)
+        # rubocop:disable HandleExceptions
+        rescue err_kls
```

Comment:
Why return output after this rather than the error object?

Reply:
because the only thing the error object is used for is the exit code. The output is where the cli actually did everything